src/tools/auth0/handlers/actions.ts

```diff
@@ -70,7 +70,7 @@ function isActionsDisabled(err) {
 export default class ActionHandler extends DefaultAPIHandler {
   existing: Asset[] | null;
 
-  constructor(options) {
+  constructor(options: DefaultAPIHandler) {
```

Comment:
The actions handler needs this type in the constructor to appropriately evaluate the types of the function overrides it is applying.